src/coreComponents/constitutive/docs/BiotPorosity.rst

```diff
@@ -0,0 +1,46 @@
+.. _BiotPorosity:
+
+
+####################################################
+Biot Porosity Model
+####################################################
+
+
+Overview
+======================
+
+Accodirng to the poroelasticity theory, the porostiy (pore volume) change with respect to pressure is expressed as the following Biot Porosity Model:
+
+.. math::
+   \frac{d{\phi}}{dp} = {\alpha}{\frac{d{\epsilon}}{dp} + \frac{\alpha - \phi}{K_s}
```

Comment:
Do we have a reference for this expression?

Reply:
no, but let me find one and add it.

Reply:
Also, does this expression reflect how the model is implemented?